server/lib/nl/common/constants.py

```diff
@@ -310,6 +318,14 @@
         "Highest temperature decrease by 2050 per RCP 4.5 (intermediate) scenario (°C)",
     "ProjectedMin_Until_2050_DifferenceRelativeToBaseDate1981To2010_Min_Temperature_RCP60":
         "Highest temperature decrease by 2050 per RCP 6.0 (slightly pessimistic) scenario (°C)",
+    "DiffRelativeToAvg_1980_2010_MaxTemp_Daily_Hist_5PctProb_Greater_Atleast1DayADecade_CMIP6_Ensemble_SSP245": 
+        "Relative to Average in 1980-2010, Predicted Max Temperature At Least Once A Decade With 5 Percent Likelihood, SSP245",
```

Comment:
I slightly like the readability of the prior titles.

Highest temperature increase by "DATE" per SSP 245 (intermediate) scenario

Reply:
The info about X% likelihood, reference period can be part of the description https://github.com/datacommonsorg/website/blob/a0dc20dcfe4ca02c76dde83f3f76338d91989887/server/lib/nl/common/topic.py#L277-L278, which can be long

Reply:
Done